src/database.py

```diff
@@ -197,8 +186,10 @@ def get_latest_forecast_values_for_a_specific_gsp_from_database(
         )
 
     # convert to pydantic objects
-    if isinstance(forecast_values[0], ForecastValueSevenDaysSQL) or isinstance(
-        forecast_values[0], ForecastValueSQL
+    if (
```

Comment:
Does this use python 3.10? Could be a good use for `match case` here if so.

Reply:
isnt match case for if, elif, elif, ...., rather than one if statment?